cfgmgr/intfmgr.cpp

```diff
@@ -843,6 +857,30 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
                 m_loopbackIntfList.insert(alias);
                 SWSS_LOG_INFO("Added %s loopback interface", alias.c_str());
             }
+
+            if (adminStatus.empty())
+            {
+                adminStatus = "up";
+            }
+            else if (adminStatus != "up" && adminStatus != "down")
```

Comment:
Check seems incorrect. Shouldn't it be `||` ?

Reply:
It should be ```&&```. If we make it ```||``` then we will end up reporting that we got an incorrect value even if the adminStatus gets passed in as ```down``` when it hits the 1st adminStatus check.